src/StatusQ/Popups/StatusSearchPopup.qml

```diff
@@ -0,0 +1,198 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.12
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Popups 0.1
+
+
+StatusModal {
+    id: root
+    width: 700
+    height: (!!searchResults && (searchResults.count >= 0) && (searchText !== "")) ? (((searchResults.count < 5)) ? 560 : 770) : 142 //970
+    anchors.centerIn: parent
+    showHeader: false
+    showFooter: false
+
+    property string searchText: contentComponent.searchText
+    property string selectionBadgeImage: ""
+    property string selectionBadgePrimaryText: ""
+    property string selectionBadgeSecondaryText: ""
+    property string selectionBadgeIdenticonColor: ""
+    property bool loading
+    property Component searchOptionsPopupMenu
+    property ListModel searchResults: ListModel { }
+    function resetSelectionBadge() {
+        selectionBadgeImage = "";
+        selectionBadgePrimaryText = "Anywhere";
+        selectionBadgeSecondaryText = "";
+        selectionBadgeIdenticonColor = "";
+    }
+
+    content: Item {
+        width: parent.width
+        height: root.height
+        property alias searchText: inputText.input.text
+
+        ColumnLayout {
+            id: contentItemColumn
+            anchors.fill: parent
+            spacing: 0
+            Item {
+                Layout.fillWidth: true
+                Layout.preferredHeight: 63
+                StatusInput {
+                    id: inputText
+                    anchors.left: parent.left
+                    anchors.leftMargin: 8
+                    anchors.right: parent.right
+                    anchors.rightMargin: 8
+                    anchors.bottom: parent.bottom
+                    input.icon.name: "search"
+                    input.placeholderText: "Search"
+                    input.bgVisible: false
+                }
+            }
```

Comment:
The size of this icon seems wrong. This is what has been designed:

<img width="778" alt="Screenshot 2021-08-03 at 23 06 56" src="https://user-images.githubusercontent.com/445106/128086203-69a8f6da-5f4a-404f-ab93-68d808ba33f8.png">

This is what it looks like:

<img width="737" alt="Screenshot 2021-08-03 at 23 07 23" src="https://user-images.githubusercontent.com/445106/128086252-9e9054c9-adc2-4588-a370-35922b9e8040.png">


If `StatusBaseInput` is the issue here and not flexible enough, it's perfectly fine to introduce a custom inline input for this scenario.

Reply:
Exactly, does it really make sense to duplicate a whole text input for just making the icon bigger or maybe make the existing StatusInput able to have a bigger icon?

Reply:
I'm honestly happy fine either way. The search input is a super special version of an input. If it can't be easily derived from `StatusBaseInput` that's perfectly fine.

I personally would probably build a new one (that's also dramatically simpler than `StatusBaseInput`. But if you prefer making the base one more flexible, please go ahead!

Reply:
Done, switched to use a TextInput instead